cibyl/sources/elasticsearch/api.py

```diff
@@ -95,16 +96,14 @@ def __query_get_hits(self: object, query: dict, index: str = '*') -> list:
         :return: List of hits.
         """
         try:
-            with self.es_client.connect() as es_connection:
-                LOG.info("Using the following query: {}"
-                         .format(str(query).replace("'", '"')))
-                hits = [item for item in scan(
-                    es_connection,
-                    index=index,
-                    query=query,
-                    size=10000
-                )]
-            es_connection.transport.close()
+            LOG.debug("Using the following query: {}"
```

Comment:
Flake8 is complaining about the indentation of the second line. With logging you can use lazy old-style formatting, like this:
```python
LOG.debug("Using the following query: %s",
          str(query).replace("'", '"')) 
```
which pylint also prefers

Reply:
I didn't have time to check it. Change done.